src/Plugins/ImageProcessor.Web/ImageProcessor.Web.Plugins.AzureBlobCache/AzureBlobCache.cs

```diff
@@ -381,11 +381,12 @@ public override void RewritePath(HttpContext context)
                         {
                             HttpResponse contextResponse = context.Response;
                             cachedStream.CopyTo(contextResponse.OutputStream);
+                            string contentType = response.ContentType;
 
                             // Mimetype can be null when returning from the cache.
                             ImageProcessingModule.SetHeaders(
                                 context,
-                                string.IsNullOrWhiteSpace(this.mimeType) ? contextResponse.ContentType : this.mimeType,
```

Comment:
@JimBobSquarePants  yes, proper fix is to get ContentType from response/azure.

the "image/xyz" is still there, but is just a fallback if Azure doesn't.